source/docs/prismic/integration-fields.md

```diff
@@ -0,0 +1,289 @@
+
+## Configure Integration fields in Prismic
+
+<blockquote class="warning">
+  Integration fields is still a beta feature in Prismic. You must request the feature activation for your repository via the Prismic [commmunity forum](https://community.prismic.io/t/feature-activations-graphql-integration-fields-etc/847) or support.
+</blockquote>
+
+You must set up Integration fields in Prismic for each Integration field registered in Front-Commerce. To do so, create a **"Custom API"** Integration field from the **"Settings > Integration Fields"** page.
+
+![Integration field creation form](./assets/integration-fields/new-integration-field.jpg)
+
+1. the **"Endpoint"** value must be the URL of the Front-Commerce Integration field API endpoint: `https://your-store.example.com/prismic/integration/{name}`
+2. the **"Access token"** value must be the value of your `FRONT_COMMERCE_PRISMIC_WEBHOOK_SECRET` environment variable ([also used for webhooks](/docs/prismic/installation.html#Configure-the-environment-for-Prismic))
+3. create the Integration field
+
+The **"Settings > Integration Fields"** page will now display the Integration field data synchronization status.
+
+Content managers can now design custom types that contain 3rd-party data from your Front-Commerce application. After the first successful synchronization, they could select values in documents without leaving the writing room.
+
+## Update your types and data in Prismic
+
+Integration fields created in the previous section should now be available in the Custom types editor. You can drag & drop them to your content model in build mode as any other field.
+
+![Custom type field from Integration field](./assets/integration-fields/configure-integration-field.jpg)
+
+See [Configure Integration Fields (Prismic documentation)](https://prismic.io/docs/core-concepts/integration-fields) for details.
+
+In the writing room, Content managers can now create or update documents to select values from Front-Commerce data for the configured fields:
+
+![Selecting a value for a document field using an Integration field](./assets/integration-fields/document-integration-field-lookup.jpg)
+
+## Expose data in your GraphQL schema
+
+This section supposes that you have already exposed data from the previously modified Prismic custom type in your GraphQL schema. It will guide you throughout the process of exposing an additional Integration field data.
+
+**We've tried to make the process as idiomatic as what you might know from implementing custom Front-Commerce features.**
+
+### Add the field to your schema
+
+First, you have to update your GraphQL schema definition with the new field to expose. Instead of exposing an id or a SKU, we will expose an existing GraphQL type for a `product` field.
```

Comment:
```suggestion
First, you have to update your GraphQL schema definition with the new field to expose. Instead of exposing an id or a SKU, we will expose an existing GraphQL type, in this case a Product for a `product` field.
```